src/sql/plot.py

```diff
@@ -509,28 +516,37 @@ def _histogram(table, column, bins, with_=None, conn=None, facet=None):
             group by 1
             order by 1;
             """
-        )
+
+        if use_backticks:
+            template_ = template_.replace('"', "`")
```

Comment:
I think in the future we can make it dynamic because there might be exceptional DBs, and we don't want to have to pass `use_backticks` and check it every time we run a query.

Reply:
I'm passing it depending on the dialect type, but yes, we need to come up with a more reliable way